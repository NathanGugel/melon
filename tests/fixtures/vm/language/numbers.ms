io.print(5e-5);
io.print(5e10);
io.print(5e10 + 5e-5);
io.print(5.456e3);