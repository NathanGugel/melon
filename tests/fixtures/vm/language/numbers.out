50000000000.000053406
5456.000000000
